### DISCUSSION TITLE
[Other] How to specify multi rabbit hosts in one amqp URI ?

### DISCUSSION THREAD

**Discussion by trikker** [rabbitmq/rabbitmq-server#12741] [Other] [answered]
### Community Support Policy

- [X] I have read [RabbitMQ's Community Support Policy](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md)

### RabbitMQ version used

3.13.7 or older

### How is RabbitMQ deployed?

Generic binary package

### Steps to reproduce the behavior in question

Hi, I want to specify multi hosts in one rabbitmq ampq URI, which is used as broker in apache airflow. But I find that it seems only one host is allowed in the URI, see docs: https://www.rabbitmq.com/docs/uri-spec , so how can I get high availability when the host is down?

Thanks for helping.

**Comment by michaelklishin** [comment, marked as answer]
@trikker this is not really an appropriate place for Apache Airflow questions. There is only one host in a URI, including AMQP 0-9-1 URIs. 

Depending on the tool or library used, you may be able specify multiple URIs or endpoints (as hosts, `host:port` pairs or something like that). This is true for multiple [AMQP 0-9-1 client libraries](https://www.rabbitmq.com/client-libraries/java-api-guide#endpoints-list), [RabbitMQ stream client libraries](https://www.rabbitmq.com/blog/2021/07/23/connecting-to-streams), [shovels](https://www.rabbitmq.com/docs/shovel#clustering), and so on.

Airflow may or may not provide a way to specify multiple URIs but this is not something Team RabbitMQ can help you with.
